install.md

```diff
@@ -252,7 +252,7 @@ A proper description of setting up CNI networking is given in the [`cni` README]
 Using the CNI plugins from above, a more basic network config is achieved with:
 
 ```bash
-mkdir -p /etc/cni/net.d
+sudo mkdir -p /etc/cni/net.d
 curl -qsSL https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist | sudo tee /etc/cni/net.d/99-loopback.conf
```

Comment:
The next line would require sudo also, since it is attempting to write to /etc/cni/net.d

Reply:
@rhatdan the "sudo tee" handles that :)

Reply:
Right I missed that

Reply:
How about:

sudo curl -qsSL -o  /etc/cni/net.d/99-loopback.conf https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist 

Then a simpleton like me, will not get fooled by the weird syntax.

Reply:
I'll make a new pull request for this.
I also noticed another issue with conman - apparently conman is not within $PATH when following the build by source.

Reply:
and I agree with you, for readability your solution is much better :)